package/src/components/CheckoutActions/v1/CheckoutActions.js

```diff
@@ -286,13 +292,13 @@ class CheckoutActions extends Component {
   };
 
   renderAction = (action, currentActiveActions) => {
-    const { components: { CheckoutAction, CheckoutActionIncomplete } } = this.props;
+    const { className, components: { CheckoutAction, CheckoutActionIncomplete } } = this.props;
     const isActive = currentActiveActions.find((_id) => _id === action.id);
     const actionStatus = isActive ? "active" : action.status;
     const { activeLabel, completeLabel, incompleteLabel } = action;
 
     return (
-      <Action key={action.id}>
+      <Action className={className} key={action.id}>
```

Comment:
It's a bit strange that this one renders a list and puts the className on every list element. At the very least I would clarify this behavior in the `className` prop docs. But it might make sense to add a wrapper div in the render function? In general, are we ok with a component that renders an array rather than a single element? @nnnnat @mikemurray @willopez

Reply:
I thought this was weird too. It's the only component where this is the case. We have no root element, which I didn't even think was possible in React.

Reply:
I had a similar situation in the `AddressBook` I ended up adding a `<div className={className}>` wrapper around the return in the main `render` method.

Reply:
I believe a wrapper should be added and the `className` applied to it, instead of to each `Action`